README.md

```diff
@@ -50,35 +50,36 @@ pip install git+https://github.com/gridsingularity/d3a-api-client.git
 
 ## How to use the Client
 
-### Interacting with the CLI
-You can run one of the setup scripts residing in the `d3a_api_client/setups` folder, or you can 
-set the path for your scripts directory as list in the `d3a_api_client.constants` .
-In order to run a module you can use the following command
-```
-d3a-api-client run --setup {module_name}
-```
+### Interacting via CLI
+In order to get help, please run:
+```
+d3a-api-client run --help
+```
+Following is the more in depth explanation of individual flags supported,
+- base-setup-path --> Sets the base path where you client script resides else `d3a_api_client/setups` is taken as default for your client-scripts. User could provided both an absolute and relative file path.
```

Comment:
```suggestion
- base-setup-path --> Sets the base path where your client script resides, otherwise `d3a_api_client/setups` is taken as default for your client scripts. Users can provide either an absolute or a relative file path.
```